### ISSUE TITLE
Does this tool supports CRM 2016?

### ISSUE THREAD

**Issue by coolsama** [Microsoft/CRM-Performance-Toolkit#21]
Hi there,
I am looking to test the performance of CRM on CRM 2016 on premise version.

Please assist,

Thanks,
Sama.

**Comment by coolsama** [comment]
Yes it works, I have managed to deploy and record the results. Great tool guys.

**Comment by jrdonta** [comment]
We used Visual Studio 2012 instead of 2013 and we do not see CRM 2016 from the drop down of toolkit.   Any suggestion ?
